modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -66,6 +66,24 @@ module_stream_test_construct (ModuleStreamFixture *fixture,
       g_assert_null (modulemd_module_stream_get_stream_name (stream));
 
       g_clear_object (&stream);
+    
+    
+      /*Test that the new() function works with no module or stream*/
```

Comment:
Please leave a space between the start and end of comments.
```suggestion
      /* Test that the new() function works with no module or stream */
```